app/routes/orders/new.js

```diff
@@ -29,10 +29,18 @@ export default class NewRoute extends Route {
     });
 
     const eventDetails = await order.query('event', { include: 'tax' });
+
+    let taxDetails;
+
+    if (eventDetails.isTaxEnabled) {
+      taxDetails = await eventDetails.get('tax', { cache: true, public: true });
```

Comment:
Multiple awaits

Reply:
sir can i write taxdetails in return like this using hash, this will also reduce multiple awaits

```
taxDetails : eventDetails.isTaxEnabled ? eventDetails.get('tax', { cache: true, public: true }) : undefined;
```
because i think the 404 error will again generate if we try to get taxDetails for those events which do not have tax details

Reply:
it is working sir